src/openscm_runner/adapters/ciceroscm_adapter/read_results.py

```diff
@@ -0,0 +1,278 @@
+"""
+Module that reads in Cicero-SCM results
+and returns data to append to SCMRun
+"""
+import os
+
+import numpy as np
+import pandas as pd
+
+# from .make_scenario_files import SCENARIOFILEWRITER
+
+
+def get_data_from_conc_file(folder, variable):
+    """
+    Get data from concentration files
+    """
+    df_temp = pd.read_csv(os.path.join(folder, "temp_conc.txt"), delimiter=r"\s+")
+    years = df_temp.Year[:]
+    timeseries = df_temp[variable].to_numpy()
+    # Todo: Deal with units
```

Comment:
Is this still TODO?

Reply:
If pint can do conversions of Pg/C etc. I think this has been dealt with now (it wasn't previously)...